src/panel/category_panel.js

```diff
@@ -52,15 +52,15 @@ export default class CategoryPanel {
   }
 
   async search() {
-    this.loading = true
+    this.loading = true;
```

Comment:
Just curiosity: why adding the semicolon when there are none in the rest of the code?

Reply:
sorry, reflex